core/src/main/java/jenkins/model/item_category/ItemCategory.java

```diff
@@ -0,0 +1,112 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import hudson.ExtensionPoint;
+import hudson.model.ModelObject;
+import jenkins.model.Messages;
+
+/**
+ * A category for {@link hudson.model.Item}s.
+ *
+ * @since TODO
+ */
+public abstract class ItemCategory implements ModelObject, ExtensionPoint {
```

Comment:
Well the intended meaning of “model object” was something stronger than that, more of a renderable top-level object with a URL and so on, though as noted in [JENKINS-26091](https://issues.jenkins-ci.org/browse/JENKINS-26091) the original API design was mangled. Unless you are actually taking advantage of this being `ModelObject` for some reason, which it does not seem you are, I would recommend not using that API, and just directly declare a method like `getDisplayName` (or simply `getName`).